vertx-pojo-mapper-common/src/main/java/de/braintags/vertx/jomnigate/init/DataStoreSettings.java

```diff
@@ -163,4 +164,23 @@ public boolean isClearDatabaseOnInit() {
   public void setClearDatabaseOnInit(boolean clearDatabaseOnInit) {
     this.clearDatabaseOnInit = clearDatabaseOnInit;
   }
+  
+  /* (non-Javadoc)
+   * @see java.lang.Object#clone()
+   */
+  @Override
+  public DataStoreSettings clone() {
```

Comment:
Sonar advices to no more use the clone because its problematic  (squid:S2975) and better to use a CopyConstructor like new DataStoreSettings( DataStoreSettings source )